spec/models/services/facebook_spec.rb

```diff
@@ -22,14 +22,46 @@
       post_params = @service.create_post_params(post)
     end
 
-    it 'does not add post link when no photos' do
+    it "adds URL of post to end if no link at end" do
       message = "Some text."
       post = double(message: double(plain_text_without_markdown: message, urls: []), photos: [])
       post_params = @service.create_post_params(post)
-      expect(post_params[:message]).not_to include "http"
+      expect(post_params[:message]).to include "(via http:"
     end
 
-    it 'sets facebook id on post' do
+    it "adds URL at the end with ending with valid HTTP url" do
```

Comment:
I don't think we need that many tests for different urls in different places in the message, because it just doesn't make any difference.

Reply:
It made a difference when I was putting the "via" link before the last URL, but you are right they are superfluous now.